petabtests/cases/v1.0.0/sbml/0020/0020.py

```diff
@@ -0,0 +1,98 @@
+from inspect import cleandoc
+
+import pandas as pd
+from petab.v1.C import *
+
+from petabtests import PetabTestCase, analytical_a, antimony_to_sbml_str
+from pathlib import Path
+
+DESCRIPTION = cleandoc("""
+## Objective
+
+This case tests handling of initial concentrations that are specified
+in the conditions table. For species `A`, the initial concentration is
+estimated. For species `B`, the initial concentration is specified as
+`NaN` in the condition table, thus the SBML model initial value should 
+be used.
+
+## Model
+
+A simple conversion reaction `A <=> B` in a single compartment, following
+mass action kinetics.
+""")
+
+# problem --------------------------------------------------------------------
+ant_model = """
+model *petab_test_0011()
+  compartment compartment_ = 1;
+  species A in compartment_, B in compartment_;
+
+  fwd: A => B; compartment_ * k1 * A;
+  rev: B => A; compartment_ * k2 * B;
+
+  A = a0;
+  B = 3;
+  a0 = 1;
+  k1 = 0;
+  k2 = 0;
+end
+"""
+model_file = Path(__file__).parent / "_model.xml"
+model_file.write_text(antimony_to_sbml_str(ant_model))
+
+condition_df = pd.DataFrame(
+    data={
+        CONDITION_ID: ["c0"],
+        "A": ["initial_A"],
+        "B": ["initial_B"],
```

Comment:
Where is `initial_B` defined? This is meant to be "NaN" (empty string) according to the test case docstring, right?